deployments/k8s/04-statefun/00-module.yaml

```diff
@@ -5,11 +5,12 @@ metadata:
   name: module-config
 data:
   module.yaml: |+
-    kind: io.statefun.endpoints.v2.http
+    kind: io.statefun.endpoints.v2/http
     spec:
       functions: example/*
       urlPathTemplate: http://functions.statefun.svc.cluster.local:8000/statefun
       transport:
+        type: io.statefun.transports.v1/okhttp
```

Comment:
This shouldn't be needed anymore once 3.2.0 was released, but apparently was the case in 3.1.
If you still have the capacity would you be able to quickly verify that this works for you without specifying the `type`?

Reply:
Hi @igalshilman!

I'm trying to get the example to work in gke using a gcs connector instead of minio at the moment. Once I get that up and running then I'll be happy to swing back and see if `type` still needs to be specified here in 3.2.0. 👍

Reply:
cool, thanks for doing is!
If you need any assistance/stumble something let us know on the user mailing list for Apache Flink.
I'd be happy to help out!

Reply:
Great news! Everything worked in Statefun 3.2.0 even when `type` wasn't specified. I've pushed a new commit removing that line from this PR. 🥳